.rubocop_todo.yml

```diff
@@ -126,6 +126,11 @@ Lint/SafeNavigationConsistency:
 Lint/ShadowingOuterLocalVariable:
   Enabled: false
 
+# Offense count: 2
+# Cop supports --auto-correct.
+Lint/UnneededCopDisableDirective:
```

Comment:
these disablings in the code should get deleted.

Reply:
AFAICT this was added (and removed from the _todo file) because of discrepancies of `rubocop` vs. `rake rubocop`. Personally I'd like to have this not in the _todo file, but I won't block the PR on this.